packages/atomic/src/components/search/atomic-field-condition/atomic-field-condition.spec.ts

```diff
@@ -0,0 +1,121 @@
+import type {Result} from '@coveo/headless';
+import {html} from 'lit';
+import {describe, expect, it} from 'vitest';
+import {renderInAtomicResult} from '@/vitest-utils/testing-helpers/fixtures/atomic/search/atomic-result-fixture';
+import {buildFakeResult} from '@/vitest-utils/testing-helpers/fixtures/headless/search/result';
+import type {AtomicFieldCondition} from './atomic-field-condition.js';
+import './atomic-field-condition.js';
+
+describe('atomic-field-condition', () => {
+  const renderFieldCondition = async ({
+    ifDefined = '',
+    ifNotDefined = '',
+    mustMatch = {},
+    mustNotMatch = {},
+    resultState = {},
+  }: {
+    ifDefined?: string;
+    ifNotDefined?: string;
+    mustMatch?: Record<string, unknown>;
+    mustNotMatch?: Record<string, unknown>;
+    resultState?: Partial<Result>;
+  } = {}) => {
+    const result = buildFakeResult(resultState);
+
+    const {element} = await renderInAtomicResult<AtomicFieldCondition>({
+      template: html`
+        <atomic-field-condition
+        if-defined="${ifDefined}"
+        if-not-defined="${ifNotDefined}"
+        .mustMatch="${mustMatch}"
+        .mustNotMatch="${mustNotMatch}">
+          <span id="condition-met">Condition Met</span>
+      </atomic-field-condition>
+      `,
+      selector: 'atomic-field-condition',
+      result,
+    });
+
+    return {
+      element,
+      content: element?.querySelector('#condition-met'),
+    };
+  };
+
+  it('should render when no conditions are defined', async () => {
```

Comment:
atomic-field-condition doesn't really "render" anything by itself... In general maybe we should say "should render its content"?